fanuc_lrmate200id_moveit_plugins/lrmate200id_kinematics/src/fanuc_lrmate200id_manipulator_ikfast_moveit_plugin.cpp

```diff
@@ -635,7 +635,7 @@ bool IKFastKinematicsPlugin::searchPositionIK(const geometry_msgs::Pose &ik_pose
                           error_code,
                           options);
 }
-    
```

Comment:
Don't make any whitespace changes in this PR - if you want, you can put them in a separate PR. Otherwise, just PR the URDF shared ptr stuff to make it super easy for the maintainers to review.

Reply:
Fixed; my editor did it automatically...